prefix_metrics.go

```diff
@@ -0,0 +1,31 @@
+package gometer
+
+// PrefixMetrics is a Metrics wrapper, that always add
+// specified prefix to counters names.
+type PrefixMetrics struct {
+	Metrics
+	prefix string
+}
+
+var _ Metrics = (*PrefixMetrics)(nil)
+
+// NewPrefixMetrics creates new PrefixMetrics that
+// wraps original Metrics and use specified prefix.
+func NewPrefixMetrics(m Metrics, prefix string) *PrefixMetrics {
```

Comment:
Let's save an ability to have a formatted prefix as we have in `Group`.

```go
func Group(format string, v ...interface{}) *CountersGroup {		
 	return &CountersGroup{		
 		prefix:   fmt.Sprintf(format, v...),		
 		counters: make(map[string]*Counter),		
 	}		
 }
```

I'd prefer to rename constructor to `WithPrefix` and rework it a bit:

```go
m := gometer.New().WithPrefix("foo.%s.baz", "bar")

// or global

m := gometer.WithPrefix("foo.%s.baz", "bar")

```

Reply:
fixed